src/auditd-config.c

```diff
@@ -1034,6 +1034,11 @@ static int space_action_parser(const struct nv_pair *nv, int line,
 				if (check_exe_name(nv->option, line))
 					return 1;
 				config->space_left_exe = strdup(nv->option);
+			} else if (failure_actions[i].option == FA_HALT) {
```

Comment:
Which one is a better approach?

- Return 1 indicating failure in case HALT is selected for space_left_action (behavior from the patch)
- Log the error via audit_msg and fallback to SYSLOG maybe?

Reply:
In all other cases we do both.